packages/blade/src/components/Icons/_Svg/Rect/__tests__/Rect.web.test.tsx

```diff
@@ -0,0 +1,14 @@
+import Rect from '..';
+import renderWithTheme from '../../../../../_helpers/testing/renderWithTheme.web';
+import Svg from '../../Svg';
+
+describe('<Rect />', () => {
+  it('should render html rect component', () => {
```

Comment:
```suggestion
  it('should render rect component', () => {
```

any reasons to be specific?

the reason I'm raising this is because this might become a pattern overtime and people might use it unnecessarily if there's no importance of this

Reply:
web test says `should render html rect component` & native tests says `should render react-native-svg Rect component`. It makes it clear where these rect components are coming from for both platforms. Any issue with adding `html`?